src/ape/utils.py

```diff
@@ -479,11 +479,19 @@ class AbstractDataClassMeta(DataClassMeta, ABCMeta):
 errors will occur. This class cannot be instantiated on its own.
 """
 
+
+def convert(*args, **kwargs):
+    from ape import convert
+
+    return convert(*args, **kwargs)
+
+
```

Comment:
I don't know if I like this being here, since you have to load a dependency from the top

Reply:
removed, I only needed this in `chain`, so I just moved it there